Doc/library/sqlite3.rst

```diff
@@ -1224,6 +1252,38 @@ Connection objects
 
       .. versionadded:: 3.11
 
+   .. attribute:: autocommit
+
+      This attribute controls :pep:`249`-compliant transaction behaviour.
+      :attr:`!autocommit` has three allowed values:
+
+      * ``False``: Select :pep:`249`-compliant transaction behaviour,
+        implying that :mod:`!sqlite3` ensures a transaction is always open.
+        Use :meth:`commit` and :meth:`rollback` to close transactions.
+
+        This is the recommended value of :attr:`!autocommit`.
+
+      * ``True``: Use SQLite's `autocommit mode`_.
+        :meth:`commit` and :meth:`rollback` have no effect in this mode.
+
+      * :data:`LEGACY_TRANSACTION_CONTROL`:
+        Pre-Python 3.12 (non-:pep:`249`-compliant) transaction control.
```

Comment:
This comment makes no sense. Both boolean values are specified by the upcoming PEP 249 amendment; leaving one of the available boolean values out of the spec would be absurd. The only thing that is not PEP 249 compliant is the legacy mode.

Reply:
I had this understanding too (both boolean values are PEP-249-compliant), but since you documented PEP-249 compliance only for `autocommit=False` in this list, I interpreted `autocommit=True` as non PEP-249-compliant in the sense that PEP-249 prescribes database drivers to use `autocommit=False` *by default*.

So since the reading confused me, I suggest to state explicitly that `autocommit=True` is PEP-249-compliant in this list.

Reply:
I'm not convinced, but if this turns out to be a problem we can address it in a follow-up issue and PR.